src/hooks/useLocalStorage.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import { Todo } from '../types/Todo';
+
+export function useLocalStorage(
+  key: string,
+  startValue: Todo[],
+): [Todo[], (value: Todo[]) => void] {
```

Comment:
not critical here, more FYI: you may do this function reusable in different cases - Use generic type instead of predefined `Todo[]`